src/SFA.DAS.EmployerApprenticeshipsService.Application/Commands/CreateAccount/CreateAccountCommandHandler.cs

```diff
@@ -66,6 +68,8 @@ public CreateAccountCommandHandler(IAccountRepository accountRepository, IUserRe
             {
                 AccountId = accountId
             });
+
+            await _mediator.SendAsync(new CreateAccountEventCommand { HashedAccountId = hashedAccountId, Event = "AccountCreated" });
```

Comment:
Should this be
`await _mediator.PublishAsync(new CreateAccountEvent());`
Where CreateAccountEvent should be IAsyncNotification